torchtext/experimental/vocab.py

```diff
@@ -6,12 +6,43 @@
 import torch.nn as nn
 from torchtext._torchtext import (
     Vocab as VocabPybind,
-    _load_vocab_from_file
+    _load_vocab_from_file,
+    _load_vocab_from_raw_text_file
 )
 
 logger = logging.getLogger(__name__)
 
 
+def vocab_from_raw_text_file_object(file_like_object, jited_tokenizer, min_freq=1, unk_token='<unk>', num_cpus=32):
+    r"""Create a `Vocab` object from a raw text file like object.
```

Comment:
I don't think this gives the functionality (such as applying a generic, JIT'd tokenizer in parallel), any justice. Also num_cpus=32 seems quite aggressive as a default, I'd set it to 4 to reflect a rough guess of what most PC can support.

Reply:
Updated the description to reflect this. Also changed num cpus to 4